pkg/ccl/backupccl/testdata/backup-restore/online-restore-in-progress-imports

```diff
@@ -1,35 +1,66 @@
-# This test ensures that online restore failes when restoring tables 
-# undergoing an in progress import
+# This test ensures that online restore generates an IMPORT ROLLBACK
+# job.
 
 reset test-nodelocal
 ----
 
 new-cluster name=s1 disable-tenant
 ----
 
+exec-sql
+CREATE DATABASE d;
+USE d;
+CREATE TABLE foo (i INT PRIMARY KEY, s STRING);
+CREATE TABLE baz (i INT PRIMARY KEY, s STRING);
+INSERT INTO baz VALUES (1, 'x'),(2,'y'),(3,'z');
+INSERT INTO foo VALUES (4, 'a');
+----
 
 exec-sql
-SET CLUSTER SETTING jobs.debug.pausepoints = 'import.after_ingest';
+EXPORT INTO CSV 'nodelocal://1/export1/' FROM SELECT * FROM baz;
+----
+NOTICE: EXPORT is not the recommended way to move data out of CockroachDB and may be deprecated in the future. Please consider exporting data with changefeeds instead: https://www.cockroachlabs.com/docs/stable/export-data-with-changefeeds
+
+
+
+exec-sql
+SET CLUSTER SETTING bulkio.backup.elide_common_prefix.enabled = false;
```

Comment:
why did you set this?

Reply:
This test was originally written at a point where getting a database require working with this option in place simply didn't work. I'd have to put together exactly when on master when that was to remember. But we can remove it now.

Reply:
Sounds good, thanks!